external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaRelationSuite.scala

```diff
@@ -158,19 +157,21 @@ abstract class KafkaRelationSuiteBase extends QueryTest with SharedSQLContext wi
     val topic = newTopic()
     testUtils.createTopic(topic, partitions = 3)
     testUtils.sendMessage(
-      topic, ("1", Array(("once", "1".getBytes), ("twice", "2".getBytes))), Some(0)
+      topic, ("1", Seq()), Some(0)
     )
     testUtils.sendMessage(
-      topic, ("2", Array(("once", "2".getBytes), ("twice", "4".getBytes))), Some(1)
+      topic, ("2", Seq(("a", "b".getBytes("UTF-8")), ("c", "d".getBytes("UTF-8")))), Some(1)
```

Comment:
I'd use `StandardCharsets.UTF_8`. If we concern about longer line, it won't hurt to import it and just use `UTF_8` in this method.

Reply:
You are right, `StandardCharsets.UTF_8` is definitely better and I also prefer it. However, as of present `StandardCharsets.UTF_8` and `"UTF-8"` are used in the whole codebase mixed up. It seems like they should be cleaned up in the other PR. @HyukjinKwon May I take it?

Reply:
I would be happy to deal with cleaning up. Just want to avoid adding more for what Spark seems to use less. (I sought with ".getBytes" and StandardCharsets.UTF_8 is used majorly.)

Reply:
Raised #25335 for cleaning up.

Reply:
Go ahead and use `UTF-8` rather than the string here